### PR TITLE
ENGDOCS-1275

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: docker-hub/image-access-management.md [hunk 1]
```diff
@@ -4,59 +4,35 @@ keywords: image, access, management
...
 - **Docker Official Images**: A curated set of Docker repositories hosted on Hub. They provide OS repositories, best practices for Dockerfiles, drop-in solutions, and applies security updates on time.
 - **Docker Verified Publisher Images**: published by Docker partners that are part of the Verified Publisher program and are qualified to be included in the developer secure supply chain. You can set permissions to **Allowed** or **Restricted**.
 - **Community Images**: Images are always disabled when Image Access Management is enabled. These images are not trusted because various Docker Hub users contribute them and pose security risks.
 
     > **Note**
     >
-    > Image Access Management is set to Disabled by default. However, member(s) of the `owners` Team in your Organization have access to all images regardless of the settings.
+    > Image Access Management is turned off default. However, members of the `owners` team in your organization have access to all images regardless of the settings.
```
Comment (ChrisChinchilla) [inline docker-hub/image-access-management.md:28]:
```suggestion
    > Image Access Management is turned off by default. However, members of the `owners` team in your organization have access to all images regardless of the settings.
```

File: docker-hub/image-access-management.md [hunk 2]
```diff
@@ -4,59 +4,35 @@ keywords: image, access, management
...
-
-Download Docker Desktop 4.0 or a later release.
-
-- [Download and install for Windows](/desktop/install/windows-install)
-- [Download and install for Mac](/desktop/install/mac-install/)
-- [Download and install for Linux](/desktop/install/linux-install)
-
-{% include configure-registry-json.md %}
+5. Select the category restrictions for your images by selecting **Allowed**.
+     Once the restrictions are applied, your members can view the Org permissions page in a read-only format.
```
Comment (ChrisChinchilla) [inline docker-hub/image-access-management.md:31]:
```suggestion
     Once the restrictions are applied, your members can view the organization permissions page in a read-only format.
```

File: docker-hub/image-access-management.md [hunk 3]
```diff
@@ -4,59 +4,35 @@ keywords: image, access, management
...
+6. Optional: To ensure that each org member uses images in a safe and secure environment, [enfore sign-in](../docker-hub/configure-sign-in.md).
 
 ## Verify the restrictions
 
-   To confirm that the restrictions are successful, have each org member pull an image onto their local computer after signing into Docker Desktop. If they are unable to sign in, they will receive an error message.
-
-   For example, if you enable Image Access Management, your members can pull an Organization Image, Docker Official Image, or Verified Publisher Image onto their local machine. If you disable the restrictions, your members can pull any image, including Community Images.
+   To confirm that the restrictions are successful, have each organization member pull an image onto their local computer after signing in to Docker Desktop. If they're don't sign in, an error message is sent.
```
Comment (ChrisChinchilla) [inline docker-hub/image-access-management.md:36]:
```suggestion
   To confirm that the restrictions are successful, have each organization member pull an image onto their local computer after signing in to Docker Desktop. If they don't sign in, they receive an error message.
```

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]
Small changes and suggestions

**Comment by aevesdocker** [comment]
Thanks @ChrisChinchilla - updated! Ready for review again

**Review by ChrisChinchilla — APPROVED** [review]

**Pull request merged**